src/main/webapp/script.js

```diff
@@ -10,4 +10,32 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function getYouTubeComments() { 
+  const dataContainer = document.getElementById('data-container');
+  const urlInput = document.getElementById('url-entry');
+  var URL = cleanseUrl(urlInput.value);
+  fetch("/YouTubeComments?url="+URL)
+    .then(response => response.json()).then((comments) =>{
+      // Generate comments
+      dataContainer.innerText = "Howdy world";
+
+      for(i = 0; i < comments.items.length; i++) {
+        dataContainer.appendChild(createYouTubeCommentElement(comments.items[i]));
+      }
+    });
+}
+
+// Extracts video id from full url
+function cleanseUrl(url) {
+  // Split web address from parameters, extract first parameter
+  var videoId = url.split("?");
```

Comment:
What if the user inputs "google.com"? For MVP we can probably keep it as assumed, but moving forward we should make sure to check if it's a vlid URL

Reply:
Agreed, we can handle this in the try/catch possibly